packages/actions/index.ts

```diff
@@ -459,10 +460,17 @@ export const getPending = () => {
     const bilateralSettlements = filterMultiTransactions(user.address, rawPendingSettlements.bilateralSettlements.map(jsonToPendingBilateral), getState())
     settleBilateral(user, bilateralSettlements, dispatch, getState)
 
+
+    const rawInviteTxs = await creditProtocol.getInviteTransactions(user.address)
+    const pendingInviteTxs = rawInviteTxs.map(tx => new InviteTransaction(tx, true))
+    console.log('RAW PENDING INVITE TRANSACTIONS: ', rawInviteTxs)
```

Comment:
:memo: I don't particularly care for spurious `console.log()` statements in code. Totally cool as a way to debug locally, but leaving them in to committed code has a bit of a smell to it -- before you know it, running the app spits a flurry of noise that can be hard to filter through. I know our tooling is configured to strip these out before compiling for production, but it's still a bit messy.

HOWEVER: If you disagree, and these logging statements are useful to you, I sincerely and whole-heartedly support that. Since there's no end-user impact of having or not having logging statements in there, my opinion in this case is not emphatically held.